### PR TITLE
Add object detection recipe

### PR DESCRIPTION
**Pull request by MichaelClifford**
This PR adds a sample recipe for a object detection demo. It includes a new model service for object detection tasks, as well as a demo application for interacting with the model service.

### CODE REVIEW

File: object_detection/client/Contianerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM registry.access.redhat.com/ubi9/python-39:latest
```
Comment (jeffmaury) [inline object_detection/client/Contianerfile]: Rename file

File: object_detection/model_server/Contianerfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+FROM registry.access.redhat.com/ubi9/python-39:latest
```
Comment (jeffmaury) [inline object_detection/model_server/Contianerfile]: Rename file

**Review by jeffmaury — CHANGES_REQUESTED** [review]

File: object_detection/client/Contianerfile (hunk 1 above)
Comment (MichaelClifford) [reply object_detection/client/Contianerfile]: thanks @jeffmaury. Fixed.

File: object_detection/README.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+```
+
+## Build and run the client application
+
+```bash
+podman build -t object_detection_client object_detection/model_client 
+```
+
+```bash
+podman run -p 8501:8501 -e MODEL_ENDPOINT=http://10.88.0.1:8000/detection object_detection_client
```
Comment (jeffmaury) [inline object_detection/README.md:47]: I think we should give instruction how to retrieve the IP address of the model service container or have them run in a pod

Comment (MichaelClifford) [reply object_detection/README.md:47]: Can we add that as a separate issue that we can address in another PR and not block on it here? All the recipe docs currently reference 10.88.0.1. So they will all need to be changed accordingly.

File: object_detection/README.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+When using a model other than the default, you will need to set the `MODEL_PATH` environment variable. Here is an example of running the model service with a local model:
+
+```bash
+ podman run -it --rm -p 8000:8000 -v <PATH/TO>/locallm/models/vision/:/locallm/models -e MODEL_PATH=models/object_detection/facebook/detr-resnet-50/ object_detection_service
+```
+
+## Build and run the client application
+
+```bash
+podman build -t object_detection_client object_detection/model_client
```
Comment (jeffmaury) [inline object_detection/README.md]:
```suggestion
podman build -t object_detection_client object_detection/client 
```

**Review by jeffmaury — CHANGES_REQUESTED** [review]

**Review by jeffmaury — APPROVED** [review]
LGTM

**Review by sallyom — APPROVED** [review]
LGTM! Ship it!

**Pull request merged**